src/end_entity.rs

```diff
@@ -153,7 +153,6 @@ impl<'a> EndEntityCert<'a> {
     /// This function must not be used to implement custom DNS name verification.
     /// Checking that a certificate is valid for a given subject name should always be done with
     /// [EndEntityCert::verify_is_valid_for_subject_name].
-    #[cfg(feature = "alloc")]
```

Comment:
are there any tests for this API that should also lose the alloc feature gate at the same time?

Reply:
> are there any tests for this API that should also lose the alloc feature gate at the same time?

Oh, good catch. Thank you. I removed the `alloc` gate from the relevant tests in `tests/integration.rs` and folded that into the commit that removed the `alloc` requirement from the feature.